odoo/addons/base/models/ir_ui_view.py

```diff
@@ -608,7 +608,7 @@ def _get_inheriting_views(self):
             WITH RECURSIVE ir_ui_view_inherits AS (
                 SELECT id, inherit_id, priority, mode, model
                 FROM ir_ui_view
-                WHERE id IN %(ids)s AND (%(where_clause)s)
```

Comment:
Instead, can we simply return an empty view when ids not available.
```
View = self.env['ir.ui.view']
if not self.ids:
    return View
```

What do you think about this, and also add a test case for this?